DATASETS.md

```diff
@@ -0,0 +1,8 @@
+# Justice40 Datasets
+
+This page contains web links to the datasets that are uploaded as part of our data pipeline, if you want to access them directly.
+
+For now, we have two datasets:
```

Comment:
We also have all the datasets that we ETL -- once we ETL and do some normalization, they are available as CSVs in S3. I've linked to them previously in the data-pipeline README because I found I was sometimes looking for those URLs (and also sharing them). Thoughts on adding those links here?

Reply:
Oh, and we also have the data as GeoJSON, which I think would be good to highlight.

Reply:
Oh oh, and I'd love to have a dedicated section or file on how to use our tile server API. That can be a separate PR though because it would probably good to include a decent amount of how-to content

Reply:
This would definitely be the place for all of that. The main goal here was to find the right structure.

I don't know much about the tile server api, but that could go here as well since it's focused on "how do I get access to the data directly?".

Reply:
@switzersc-usds @esfoobar-usds I think if we think this file is a good idea, we should agree on an update frequency, and reflect that in a github action. What do you think of that?

For now, I'll put "whenever we run our pipeline, which we currently don't run on any specific cadence". But I think for people to find this useful, we might want to provide those guarantees (although that's out of scope of this PR I think).

Reply:
Also, since it's in s3 and s3 is not a real filesystem, we can't just send people to a directory and say "here are all the datasets we have available". They just get access denied unless we link to specific objects. Just a note that this will make keeping this up to date a bit more active effort.

Reply:
Agreed on automating the updating of this file with a Github action - that should be pretty doable. I can make a new issue for that. 

Re: S3 links, yeah, good point. We should be able to automate the link generation in the aforementioned Github Action though, since they follow a standard pattern.

Reply:
Ok, I'll let you resolve this comment when that issue exists. I like the idea of auto-generating the links somehow, and maybe making a pull request to update them automatically if they change.